functions/firebase/src/main/java/com/example/functions/firebase/FirebaseFirestoreReactive.java

```diff
@@ -70,7 +70,7 @@ public void accept(String json, Context context) throws RuntimeException {
 
     LOGGER.info(String.format("Replacing value: %s --> %s", currentValue, newValue));
     try {
-      firestore.document(affectedDoc).set(newFields, SetOptions.merge()).get();
+      FIRESTORE.document(affectedDoc).set(newFields, SetOptions.merge()).get();
     } catch (ExecutionException | InterruptedException e) {
       throw new RuntimeException(e);
```

Comment:
nit: I thought we had settled on logging this over wrapping in RuntimeException (@lesv FYI)

Reply:
Will fix this more broadly.